astropy/cosmology/flrw/tests/test_lambdacdm.py

```diff
@@ -307,6 +309,145 @@ def test_ogamma():
     assert u.allclose(cosmo.comoving_distance(z), targvals, rtol=1e-5)
 
 
+@pytest.mark.skipif(not HAS_SCIPY, reason="test requires scipy")
+def test_flat_open_closed_icosmo():
+    """Test against the tabulated values generated from icosmo.org
+    with three example cosmologies (flat, open and closed).
+    """
+
+    cosmo_flat = """\
```

Comment:
This looks like some hard-coded data. Could this be moved to a datafile?

Reply:
Good suggestions. I'll look into this!

Reply:
Done!

Reply:
I moved the data to ecsv files in ``astropy/cosmology/flrw/test/data`` and parametrized the test to cycle through the files.

Reply:
I did the data file creation as a separate commit, which directly follows the commit moving the test.

Reply:
I can squash it in when the PR is approved.